romancal/photom/tests/test_photom.py

```diff
@@ -266,19 +266,24 @@ def test_photom_step_interface(instrument, exptype):
     else:
         assert result.meta.cal_step.photom == "SKIPPED"
 
+    # Run photom correction step with reffile as N/A
+    result = PhotomStep.call(wfi_image_model, override_photom="N/A")
+
+    assert result.meta.cal_step.photom == "SKIPPED"
+
 
 @pytest.mark.parametrize(
     "instrument, exptype",
     [
         ("WFI", "WFI_PRISM"),
     ],
 )
-@pytest.mark.skipif(
-    os.environ.get("CI") == "true",
-    reason=(
-        "Roman CRDS servers are not currently available outside the internal network"
-    ),
-)
+# @pytest.mark.skipif(
+#    os.environ.get("CI") == "true",
+#    reason=(
+#        "Roman CRDS servers are not currently available outside the internal network"
+#    ),
+# )
```

Comment:
remove comments

Reply:
I think all the changes are in and I've removed all skips for the CRDS server issue since we now have  a public server.